### ISSUE TITLE
Chess clocks cause the play area to shake

### ISSUE THREAD

**Issue by atkeranov** [throneteki/throneteki#3038]
Using chess clocks in a game will cause the screen to "shake" left and right when the chess clock icon changes from one player to another.
The issue is caused by the XX:XX H1 text combined with the chess clock icon being wider than the plot area part of the screen. I believe the limit is 101px and the chess clocks are a couple of pixels wider.

**Comment by EvadLhorg** [comment]
Hi! Thank you for the report, i will look into it in the near future! I wasn´t able to see this on my first try using firefox, but had another player tell me about this issue when using Chrome. Are you also using Chrome? If not, did you experience the same issue using Firefox?

**Comment by atkeranov** [comment]
Oh, I forgot to mention which browser I am using. I initially tested with Chrome.
However, I just made a test with the latest version of Firefox and the behavior is the same. I tested with a range of zoom levels from 100% to 130% and the issue is still reproduced.
To be precise, the "shake" is very apparent when the initial timeout of 3 seconds is shown and then hidden before the actual clock starts ticking.
